modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/Checkbox.es.js

```diff
@@ -63,11 +63,12 @@ const Switcher = ({
 							{showLabel && label}
 
 							{required && (
-								<ClayIcon
-									className="reference-mark"
-									spritemap={spritemap}
-									symbol="asterisk"
-								/>
+								<span className="ddm-label-required reference-mark">
+									<ClayIcon
+										spritemap={spritemap}
```

Comment:
I believe there's no need to pass `spritemap` anymore. It's legacy code from previous versions of Clay